src/Development/Shake/Progress.hs

```diff
@@ -299,7 +298,7 @@ jsonObject xs = "{" ++ intercalate ", " [show a ++ ":" ++ b | (a,b) <- xs] ++ "}
 xterm :: Bool
 xterm = unsafePerformIO $
     -- Terminal.app uses "xterm-256color" as its env variable
-    catch_ (fmap ("xterm" `isPrefixOf`) $ getEnv "TERM") $
+    catch_ (("xterm" `isPrefixOf`) <$> getEnv "TERM") $
```

Comment:
It's now got redundant brackets - you can flip the `isPrefixOf` to `isPrefixOf "xterm"`.